frontend/packages/app/src/clutch.config.js

```diff
@@ -46,4 +46,41 @@ module.exports = {
       },
     },
   },
+  "@clutch-sh/experimentation": {
+    listExperiments: {
+      trending: true,
+      componentProps: {
+        columns: ["identifier", "targets", "type", "description"],
+        experimentTypes: {
+          "type.googleapis.com/clutch.chaos.serverexperimentation.v1.TestSpecification": {
```

Comment:
ideally, this should be a constant that lives inside of a server experimentation module and that could be imported here but I couldn't make importing work per my comment in [here](https://github.com/lyft/clutch/pull/239/files#r469641669).